blueflood-http/src/main/java/com/rackspacecloud/blueflood/http/HttpResponder.java

```diff
@@ -34,6 +36,10 @@ public static void respond(ChannelHandlerContext ctx, HttpRequest req, HttpRespo
     }
 
     public static void respond(ChannelHandlerContext ctx, HttpRequest req, HttpResponse res) {
+
+        // set response headers
+        res.setHeader("Access-Control-Allow-Origin", Configuration.getInstance().getStringProperty(CoreConfig.CORS_ALLOWED_ORIGINS));
```

Comment:
@VinnyQ , here, instead of always calling ```res.setHeader()```, we should check the value of the Config. Only call ```res.setHeader()``` if the value of the CORS_ALLOWED_ORIGINS is not empty